docs/api/public_php_api_managing_content.md

```diff
@@ -201,3 +201,57 @@ $copyUpdateStruct->identifier = $newIdentifier;
 $this->contentTypeService->updateContentTypeDraft($copyDraft, $copyUpdateStruct);
 $this->contentTypeService->publishContentTypeDraft($copyDraft);
 ```
+
+## Calendar events
+
+### Getting events
+
+To get a list of all events for a specified time period, use the `CalendarService::getEvents` method.
```

Comment:
1) IMHO `\EzSystems\EzPlatformCalendar\Calendar\CalendarServiceInterface` should be mentioned here

2) 

```suggestion
To get a list events for a specified time period, use the `CalendarService::getEvents` method.
```

because you will get `\EzSystems\EzPlatformCalendar\Calendar\EventQuery::$count` events, not all of them :wink: